pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
```

Comment:
I don't think this should be the first example. In fact, it could probably be the last one.